README.md

```diff
@@ -111,4 +111,15 @@ dotnet ef database update LastGoodMigration
 
 Download MYSQL Workbench from [here](https://dev.mysql.com/get/Downloads/MySQLGUITools/mysql-workbench-community-8.0.17-winx64.msi)
 
-Open [cslabs-db-diagram.mwb](./cslabs-db-diagram.mwb) in the root of this project using mysql workbench.
\ No newline at end of file
+Open [cslabs-db-diagram.mwb](./cslabs-db-diagram.mwb) in the root of this project using mysql workbench.
+
+
+### Development
+
+#### Git Workflow
+
+The main branch is master, which will hold the version currently running in production.
+When some work needs to be done, you will branch off from master using the
```

Comment:
@Cliftonz Are you wanting people to branch out from master? This might cause more merge conflicts but can allow changes to be deployed separately easier.

Reply:
I had this saying to branch off from dev at first, but Zac asked me to change it to branching off from master.